### DISCUSSION TITLE
sonr-findbugs support for SonarQube 9.x latest

### DISCUSSION THREAD

**Discussion by yokum3** [spotbugs/sonar-findbugs#396] [General]
Hi Team,
 Is there any plan or tentative date for sonar-findbugs plugin to support SonarQube 9.x series ?

**Comment by gtoison** [comment]
Hello,
The plugin is compatible with SonarQube 9.x since version 4.0.4 as detailed in the compatiblility table here: https://github.com/spotbugs/sonar-findbugs
Note however that this version has an issue for SonarQube installations that had version 3.3 (or earlier) of the plugin and upgraded progressively to version 4.0.4 (see: https://github.com/spotbugs/sonar-findbugs/issues/387), so better test before upgrading.
A fix for this issue is in the works and should be released in a new version soon

**Comment by 50james** [comment]
Please how can I spam someone a bug on whataspp

**Reply by gtoison** [reply]
Not sure what you're asking but whatsapp connectivity is a beyond the scope of this project
